picoscope/ps4000.py

```diff
@@ -142,6 +142,13 @@ def __init__(self, serialNumber=None, connect=True):
             self.lib = windll.LoadLibrary(str(self.LIBNAME + ".dll"))
 
         super(PS4000, self).__init__(serialNumber, connect)
+        # check to see which model we have and use special functions if needed
+        if self._lowLevelGetUnitInfo(3) == '4262':
```

Comment:
You should `unit_number = self.getUnitInfo('VariantInfo')`.  I know it is "inefficient", but i is much more readable.